static/sass/partials/_header.scss

```diff
@@ -15,23 +33,55 @@ header p.headline {
   }
 }
 
-@media only screen and (min-width: 0px) {
+
+@media only screen and (min-width: 0px) and (max-width: 289px) {
   header .camp-logo {
+      height: 500px;
+      background: url("../images/camp_logo_mobile.png") center no-repeat;
+
+    img { 
+
+      margin-left: $logo-smallest-margin-left;
```

Comment:
I don't think that's the best way to position the image. A variable is useful if it's re-used. In this case, I don't think we will re-use all of that. You can use your % straight away.